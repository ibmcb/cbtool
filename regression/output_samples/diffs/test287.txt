 status: Waiting for vm_61 (cloud-assigned uuid 940945FD-AD02-5AB2-B123-AFB9C491292C) to start...
 status: Waiting for vm_62 (cloud-assigned uuid 19D95338-EDB9-5AEC-80A0-EDD3985BF729) to start...
 status: Waiting for vm_64 (cloud-assigned uuid 99F4806F-31A9-512E-BB3D-CFA66031B8F4) to start...
 status: Waiting for vm_63 (cloud-assigned uuid CDDA8898-CC60-5842-875F-070D273DAF67) to start...
 status: Waiting for vm_65 (cloud-assigned uuid F08A8F4F-CE7E-58EA-A188-104663EE8EAE) to start...
 status: Trying to establish network connectivity to vm_61 (cloud-assigned uuid 940945FD-AD02-5AB2-B123-AFB9C491292C), on IP address 92.57.191.109...
 status: Trying to establish network connectivity to vm_63 (cloud-assigned uuid CDDA8898-CC60-5842-875F-070D273DAF67), on IP address 65.113.237.140...
 status: Trying to establish network connectivity to vm_62 (cloud-assigned uuid 19D95338-EDB9-5AEC-80A0-EDD3985BF729), on IP address 152.46.196.83...
 status: Trying to establish network connectivity to vm_64 (cloud-assigned uuid 99F4806F-31A9-512E-BB3D-CFA66031B8F4), on IP address 190.99.161.23...
 status: Trying to establish network connectivity to vm_65 (cloud-assigned uuid F08A8F4F-CE7E-58EA-A188-104663EE8EAE), on IP address 3.107.156.52...
 status: Bypassing the sending of a copy of the code tree to vm_64 (190.99.161.23)...
 status: Bypassing the sending of a copy of the code tree to vm_65 (3.107.156.52)...
 status: Bypassing the sending of a copy of the code tree to vm_61 (92.57.191.109)...
 status: Bypassing the sending of a copy of the code tree to vm_62 (152.46.196.83)...
 status: Bypassing the sending of a copy of the code tree to vm_63 (65.113.237.140)...
AI object 382FEAF1-2E9F-5F55-BD08-C4766BC63930 (named "ai_12") sucessfully attached to this experiment. It is ssh-accessible at the IP address 92.57.191.109 (940945FD-AD02-5AB2-B123-AFB9C491292C.simcloud.com).
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |83.212.20.114   |1374526402     |none                                    |3E0E082E-3CC9-58D2-96AF-2F6559324443
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |218.3.95.25     |1374526442     |none                                    |4FD4E473-B901-535E-81B0-B1D8E6D698DB
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |86.43.50.187    |1374526461     |none                                    |F8B6C5F5-69E7-54E3-ABA6-76C0E27513C8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |40.164.186.82   |1374526548     |none                                    |65F2A248-D223-51E0-9BEC-27A2FC1F0ADF
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |207.8.217.196   |1374526554     |none                                    |8CA2C11D-7925-5F77-BEC6-B769F7254F4A
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |81.175.206.221  |1374526560     |none                                    |8C6A5E67-86B3-5B4B-8CC0-FAB15590DC8B
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |92.155.71.180   |1374526566     |none                                    |4DA18983-5555-508D-9A68-DC3995B55B09
|ai_12     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |92.57.191.109   |1374526571     |none                                    |382FEAF1-2E9F-5F55-BD08-C4766BC63930
|vm_33         |client_daytrader    |nano32      |83.212.20.114   |simhostd1                  |LG                  |ai_3    |none       |6FB95A0B-DC69-5BCA-83D4-EF6A5FDE7418
|vm_34         |was                 |iron32      |141.110.188.78  |simhostb2                  |SUT                 |ai_3    |none       |E679191D-E766-5120-B1AC-B359C5F48C44
|vm_35         |db2                 |copper32    |111.33.159.57   |simhostb4                  |SUT                 |ai_3    |none       |508E8790-2816-5B0E-B5EA-25A85B4E3E5B
|vm_36         |client_daytrader    |nano32      |218.3.95.25     |simhostd4                  |LG                  |ai_4    |none       |8262E811-541A-5386-8790-420D769BCF89
|vm_37         |was                 |iron32      |167.6.69.81     |simhostb1                  |SUT                 |ai_4    |none       |4DEF67D9-E330-5E87-8DA1-E1A5C8060BF7
|vm_38         |db2                 |copper32    |87.201.50.211   |simhostb0                  |SUT                 |ai_4    |none       |57F691F0-F925-5665-BF65-1D8523FE67EC
|vm_39         |netclient           |platinum64  |86.43.50.187    |simhosta0                  |SUT                 |ai_5    |none       |C3B97918-FB0A-5D60-B179-C521859C2641
|vm_40         |netserver           |iron32      |14.75.181.209   |simhostc2                  |SUT                 |ai_5    |none       |4CEAF1BD-1B5A-541D-B8DB-84C44699AADE
|vm_48         |lb                  |iron32      |65.48.235.103   |simhostb3                  |SUT                 |ai_8    |none       |D2149426-231B-5104-A868-4088453D2CE8
|vm_47         |was                 |iron32      |16.243.58.9     |simhostc4                  |SUT                 |ai_8    |none       |B9BDE08B-E66D-56A3-8C96-E961EDCF150B
|vm_49         |client_daytrader    |nano32      |40.164.186.82   |simhostd0                  |LG                  |ai_8    |none       |4AC7923E-F7FF-58BC-B716-E3E49B9B9AFC
|vm_50         |was                 |iron32      |222.18.150.104  |simhosta3                  |SUT                 |ai_8    |none       |8ADFD3A8-2392-5ED7-88C7-3AD3E30CC079
|vm_51         |db2                 |gold32      |125.211.95.133  |simhostc4                  |SUT                 |ai_8    |none       |E090FBC3-56D8-5DB3-B4BB-B61B6E2FE270
|vm_52         |client_daytrader    |nano32      |207.8.217.196   |simhostd0                  |LG                  |ai_9    |none       |7B86C78E-A1A7-5317-A634-98BB5A9D84D4
|vm_53         |was                 |iron32      |223.39.210.221  |simhostb0                  |SUT                 |ai_9    |none       |B6CB93B7-1C28-58F9-B9E6-31662BBF9083
|vm_54         |db2                 |copper32    |159.90.112.187  |simhostb1                  |SUT                 |ai_9    |none       |695E187B-829F-5425-B2F1-D68302DC4C4F
|vm_55         |client_daytrader    |nano32      |81.175.206.221  |simhostd1                  |LG                  |ai_10   |none       |1E77B6CB-09DA-59F0-ABEC-4973A790D91E
|vm_56         |db2                 |copper32    |77.44.214.248   |simhostb3                  |SUT                 |ai_10   |none       |AE094B27-927E-5C57-9651-45E4E70E0EAC
|vm_57         |was                 |iron32      |111.184.121.1   |simhostc2                  |SUT                 |ai_10   |none       |1121B6B8-15DC-57C1-B521-AAF61E498FD0
|vm_58         |client_daytrader    |nano32      |92.155.71.180   |simhostd2                  |LG                  |ai_11   |none       |48777964-6D45-597C-A667-BE6228D7FB07
|vm_59         |was                 |iron32      |129.39.11.231   |simhostc3                  |SUT                 |ai_11   |none       |0F709DC4-4044-522C-8C4C-C5530800B6FF
|vm_60         |db2                 |copper32    |79.35.254.33    |simhostb3                  |SUT                 |ai_11   |none       |79F46801-2AEB-5031-99D0-81CD10F49106
|vm_61         |client_daytrader    |nano32      |92.57.191.109   |simhostd3                  |LG                  |ai_12   |none       |AA75E426-319B-56D3-A2AE-8BB43B2EF43E
|vm_63         |lb                  |iron32      |65.113.237.140  |simhostb2                  |SUT                 |ai_12   |none       |1F8DD11D-D6C3-5D7A-8E88-6B88F0F72CDD
|vm_62         |was                 |iron32      |152.46.196.83   |simhostc0                  |SUT                 |ai_12   |none       |B76A6630-BE91-5013-B8F6-BA7B0B08C1A5
|vm_64         |was                 |iron32      |190.99.161.23   |simhosta1                  |SUT                 |ai_12   |none       |CCC44485-54A4-5FA9-8896-9AF97FDF2B7A
|vm_65         |db2                 |gold32      |3.107.156.52    |simhostc0                  |SUT                 |ai_12   |none       |0077A9D2-8A15-56AE-8AB2-124FDD1B3CE7
|Used Memory                                          |41.47M                        
|Uptime (in seconds)                                  |13565352                      
|Total Connections Received                           |6494908                       
|Total Commands Processed                             |93843589                      
|Number of Keys                                       |656                           
|Storage Size                                         |16112607216.0                 
|Data Size                                            |13823132064.0                 
|Index Size                                           |550318384                     
|Average Object Size                                  |6602.20959797                 
|Collections                                          |60                            
|59038ADB-0C9D-5DC8-8D0F-858C40A339D8 (simzone_b) VM RESERVATIONS|10                            
|64F2E709-607C-5D1E-9ED5-402265D29FB4 (simzone_d) VM RESERVATIONS|7                             
|538D06C6-8C24-51C6-AC34-D9ECDD4133D6 (simzone_c) VM RESERVATIONS|7                             
|CA6252C4-6803-5C5C-BAA8-5BC509EE3CE0 (simzone_a) VM RESERVATIONS|3                             
