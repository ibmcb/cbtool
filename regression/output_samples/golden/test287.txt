 status: Waiting for vm_61 (cloud-assigned uuid 084557AF-1886-5DE0-B60A-10BCA6AEC8A1) to start...
 status: Waiting for vm_64 (cloud-assigned uuid C907C6DF-5430-5871-8868-17F4965B9867) to start...
 status: Waiting for vm_62 (cloud-assigned uuid AAB1D975-D58A-52FB-B302-49994FD135D1) to start...
 status: Waiting for vm_63 (cloud-assigned uuid FB0A7F33-A576-5983-B514-AE1F9F8514E4) to start...
 status: Trying to establish network connectivity to vm_63 (cloud-assigned uuid FB0A7F33-A576-5983-B514-AE1F9F8514E4), on IP address 32.74.70.10...
 status: Trying to establish network connectivity to vm_62 (cloud-assigned uuid AAB1D975-D58A-52FB-B302-49994FD135D1), on IP address 219.206.192.90...
 status: Trying to establish network connectivity to vm_64 (cloud-assigned uuid C907C6DF-5430-5871-8868-17F4965B9867), on IP address 160.94.104.59...
 status: Trying to establish network connectivity to vm_61 (cloud-assigned uuid 084557AF-1886-5DE0-B60A-10BCA6AEC8A1), on IP address 192.201.187.120...
 status: Bypassing the sending of a copy of the code tree to vm_64 (160.94.104.59)...
 status: Bypassing the sending of a copy of the code tree to vm_61 (192.201.187.120)...
 status: Bypassing the sending of a copy of the code tree to vm_62 (219.206.192.90)...
 status: Bypassing the sending of a copy of the code tree to vm_63 (32.74.70.10)...
 status: Waiting for vm_65 (cloud-assigned uuid 616EDCFF-ABF9-5C3F-BDAB-24342C0304F8) to start...
 status: Trying to establish network connectivity to vm_65 (cloud-assigned uuid 616EDCFF-ABF9-5C3F-BDAB-24342C0304F8), on IP address 29.22.190.28...
 status: Bypassing the sending of a copy of the code tree to vm_65 (29.22.190.28)...
 status: Bypassing generic VM post_boot configuration.
 status: Bypassing application-specific "setup" operations
AI object C536D81E-D85D-57E3-A40C-C6B2A7AD05C6 (named "ai_12") sucessfully attached to this experiment. It is ssh-accessible at the IP address 192.201.187.120 (084557AF-1886-5DE0-B60A-10BCA6AEC8A1.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |141.98.169.121  |1358912185     |none                                    |CECA49F8-7F0A-5A2A-88AB-6FBD3C13A188
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |20.62.246.201   |1358912222     |none                                    |AA2AF411-5127-5D04-935C-861E9991D9CD
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |88.182.228.241  |1358912240     |none                                    |22D578AD-237A-5FD0-BF2E-8526B27365A8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |120.99.243.188  |1358912324     |none                                    |DA622B54-93C1-584C-A13C-A7741CB76ADD
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |88.118.248.29   |1358912327     |none                                    |DA3C582D-82E4-5D44-87E8-6543B22AB272
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |112.27.118.248  |1358912330     |none                                    |CEE9035A-0ECD-5ACA-AC70-99DCCEA9A76C
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |127.64.103.51   |1358912333     |none                                    |E5875E74-EFDB-546D-80B6-D5BE50F7D53B
|ai_12     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |192.201.187.120 |1358912338     |none                                    |C536D81E-D85D-57E3-A40C-C6B2A7AD05C6

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |svm_stub_vmc     |uuid
|vm_33         |client_daytrader    |nano32      |141.98.169.121  |simhostd1                  |LG                  |ai_3    |none       |none             |B3C70489-FAA9-51AC-93D3-4CD8824F4C4E
|vm_35         |db2                 |copper32    |6.33.25.203     |simhostc2                  |SUT                 |ai_3    |none       |none             |7EA11E39-05B7-55AC-9DCF-859A3ED6B774
|vm_34         |was                 |iron32      |122.233.44.250  |simhostb1                  |SUT                 |ai_3    |none       |none             |C5A52973-885F-5E10-BB14-312B7C7A42EC
|vm_36         |client_daytrader    |nano32      |20.62.246.201   |simhostd0                  |LG                  |ai_4    |none       |none             |5B0F2AA5-F9AD-59EA-BC73-1158F3CE045A
|vm_38         |db2                 |copper32    |94.33.53.77     |simhostc0                  |SUT                 |ai_4    |none       |none             |1D4ACB1F-496B-5AD7-AC7B-F48747946C30
|vm_37         |was                 |iron32      |209.154.30.29   |simhosta0                  |SUT                 |ai_4    |none       |none             |61D04C3E-05A5-58A9-90A7-0828A592F247
|vm_39         |netclient           |platinum64  |88.182.228.241  |simhosta1                  |SUT                 |ai_5    |none       |none             |20413B55-E05D-5FE7-BE95-75BB5E3B3EAA
|vm_40         |netserver           |iron32      |179.252.55.59   |simhostb4                  |SUT                 |ai_5    |none       |none             |E2836789-BFD8-56CC-89C2-515EDC558C26
|vm_48         |lb                  |iron32      |10.177.191.53   |simhostb2                  |SUT                 |ai_8    |none       |none             |83D03AFF-D1C0-5B18-84FA-8AC0F0E4E141
|vm_47         |client_daytrader    |nano32      |120.99.243.188  |simhostd3                  |LG                  |ai_8    |none       |none             |4ABA46E4-AA10-5094-BBA9-0F2D7DF457B0
|vm_50         |was                 |iron32      |239.137.217.196 |simhostc1                  |SUT                 |ai_8    |none       |none             |A8F030D6-28EE-598F-920B-BFB308B2FFA4
|vm_49         |was                 |iron32      |38.98.80.56     |simhostc0                  |SUT                 |ai_8    |none       |none             |AB15F359-12EB-529E-ABC2-5A1D520E4AA5
|vm_51         |db2                 |gold32      |193.150.194.155 |simhosta3                  |SUT                 |ai_8    |none       |none             |9F49A85B-F382-56DD-8C4A-A9FF15894397
|vm_52         |client_daytrader    |nano32      |88.118.248.29   |simhostd3                  |LG                  |ai_9    |none       |none             |68465B66-D241-5522-878F-F9C38DFDFBF2
|vm_53         |was                 |iron32      |79.105.170.135  |simhostc0                  |SUT                 |ai_9    |none       |none             |7DDD2C72-498A-5216-9AFF-F46656E5DB8C
|vm_54         |db2                 |copper32    |105.2.132.123   |simhostc4                  |SUT                 |ai_9    |none       |none             |870B0B43-85AC-5DEC-A725-5F057A1DF9F6
|vm_55         |client_daytrader    |nano32      |112.27.118.248  |simhostd3                  |LG                  |ai_10   |none       |none             |B3147FAC-13F6-57E8-B1E3-8D6C2C9F6C71
|vm_56         |was                 |iron32      |24.149.93.124   |simhostc0                  |SUT                 |ai_10   |none       |none             |457C0312-7579-52BF-A901-7349AE435981
|vm_57         |db2                 |copper32    |44.239.140.109  |simhostb3                  |SUT                 |ai_10   |none       |none             |B563AF56-FD90-5545-AEE4-A1E42A62E40A
|vm_58         |client_daytrader    |nano32      |127.64.103.51   |simhostd3                  |LG                  |ai_11   |none       |none             |6EC08E6D-952F-5A24-97AB-11CC49E55BFB
|vm_59         |was                 |iron32      |118.3.19.218    |simhostc1                  |SUT                 |ai_11   |none       |none             |BD51BD8C-EDFC-5847-8357-85D22499CD94
|vm_60         |db2                 |copper32    |198.69.239.253  |simhostc1                  |SUT                 |ai_11   |none       |none             |F6FD7741-9047-5CDF-870F-AAE9508CAEB4
|vm_61         |client_daytrader    |nano32      |192.201.187.120 |simhostd0                  |LG                  |ai_12   |none       |none             |44380C9F-1399-55DD-954F-7CC55A49965C
|vm_63         |was                 |iron32      |32.74.70.10     |simhostb4                  |SUT                 |ai_12   |none       |none             |83A272D4-B991-5B5C-8C63-D3F38CB7E013
|vm_62         |lb                  |iron32      |219.206.192.90  |simhostc3                  |SUT                 |ai_12   |none       |none             |F799F109-87B5-508C-9CEC-29B6FF051041
|vm_64         |was                 |iron32      |160.94.104.59   |simhosta0                  |SUT                 |ai_12   |none       |none             |1AD4E8D6-4012-5192-9EC2-1462A5E0F97F
|vm_65         |db2                 |gold32      |29.22.190.28    |simhosta3                  |SUT                 |ai_12   |none       |none             |9B158893-FD0B-515E-B819-F7556B3BDD5C

The AI object ai_12, attached to this experiment, has the following attributes (Cloud TESTCLOUD) :
|attribute (AI object key)               |value                                
|load_balancer                           |true                                 
|sut                                     |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |3.57M                         
|Redis Version                                        |2.6.7                         
|Uptime (in seconds)                                  |459656                        
|Total Connections Received                           |2820                          
|Total Commands Processed                             |4311906                       
|Number of Keys                                       |654                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.2.2                         
|Storage Size                                         |950272                        
|Data Size                                            |238524                        
|Index Size                                           |294336                        
|Average Object Size                                  |450.045283019                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |27                            
|SVMs                                                 |0                             
|AIs                                                  |8                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |27                            
|VMs ARRIVED                                          |62                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |8                             
|AIs ARRIVED                                          |12                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |204                           

 ---------------- VMC-WIDE COUNTERS ----------------
|66C80F72-DB30-5A63-B169-31C6A86D021F (simzone_c) VM RESERVATIONS|10                            
|D3E36C93-1427-5D86-BC91-CB0FA4A8E918 (simzone_d) VM RESERVATIONS|7                             
|FEBD66A7-E012-534C-A431-00CE123E50EF (simzone_a) VM RESERVATIONS|5                             
|9859398A-822B-50DC-8C37-7E3C5CBAE602 (simzone_b) VM RESERVATIONS|5                             

